src/Cache.js

```diff
@@ -62,11 +62,13 @@ class Cache {
    * remove from cache after maxAge time has passed
    *
    * @param {*} key the key to remove
-   * @param {number} maxAge the time in milliseconds to wait before removing key
+   * @param {number} maxAge the time in milliseconds to wait before removing the key
+   * @param {Function} onExpiration a callback that is called after removing the key
    */
-  expireAfter(key: any, maxAge: number) {
+  expireAfter(key: any, maxAge: number, onExpiration: ?Function) {
     setTimeout(() => {
       this.remove(key);
+      if (onExpiration) onExpiration(key.key);
```

Comment:
Why not use both prettier and eslint together? I personally prefer [prettier-eslint-cli](https://github.com/prettier/prettier-eslint-cli), but there is also [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier) for example.

prettier-eslint is great because it allows to have a strict eslint config and doesn't require any additional configuration as it reads rules it needs from the eslintrc file. You can easily replace `lint:fix` with it.